compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/window/window/WindowStateTest.kt

```diff
@@ -659,7 +659,9 @@ class WindowStateTest {
     }
 
     @Test
-    fun `set window size by its content`() = runApplicationTest(useDelay = isLinux) {
+    fun `set window size by its content`() = runApplicationTest {
+        assumeTrue(!isLinux)
```

Comment:
Shouldn't we just "mute" it in TeamCity?

Reply:
After I spent 3 hours trying to understand why a test that should fail "passes" on TeamCity, I strongly prefer we disable tests in the code, not in TeamCity.